lib/keyring_liberator.rb

```diff
@@ -21,12 +21,16 @@ def self.get_keyring(path)
     end
 
     def self.get_keyring_named(name)
-      get_all_keyrings.find { |k| k.name == name }
+      get_all_keyrings.find { |k| k.name == name && k.path == Dir.getwd }
```

Comment:
I think this is something that we need, but as you said, this is breaking backwards compatibility. Can we make it an optional check? And from here forward we also add the path?